README.md

```diff
@@ -23,4 +23,3 @@ Everything should be good to go now. You should be able to access the webserver
 
 Also, in order to run the tests, just use the command: `./test.sh`.
 
-Full installation instructions are available in [INSTALL.md](https://github.com/metabrainz/messybrainz-server/blob/master/INSTALL.md) file.
```

Comment:
Sorry about this, forgot about it in the docker PR, it caused a conflict during rebase so I fixed it here.